src/core/blockchain/types/config.ts

```diff
@@ -11,9 +12,12 @@ export interface IBlockchainConfig {
     derivationType: DerivationType;
     coin: string;
     defaultUnit: string;
+    defaultChainId: string;
     iconComponent: React.ComponentType<any>;
     tokens: {
-        [symbol: string]: ITokenConfig;
+        [chainId: string]: {
```

Comment:
this overcomplicates everything at core/blockchain level. I'm afraid we will have to revert this :( 
So let's keep here only native tokens, that will be the same on all blockchains. With a new PR we will handle the list of default tokens list by adding them in redux token config.